commands/scan/scan.go

```diff
@@ -249,6 +251,23 @@ func (scanCmd *ScanCommand) Run() (err error) {
 	scanResults.XrayVersion = xrayVersion
 	scanResults.ScaResults = []xrutils.ScaScanResult{{XrayResults: flatResults}}
 
+	scanResults.ExtendedScanResults.EntitledForJas, err = xrutils.IsEntitledForJas(xrayManager, xrayVersion)
+	errGroup := new(errgroup.Group)
+	if scanResults.ExtendedScanResults.EntitledForJas {
+		// Download (if needed) the analyzer manager in a background routine.
+		errGroup.Go(xrutils.DownloadAnalyzerManagerIfNeeded)
+	}
```

Comment:
We already have requests to add flags to enable/disable certain JAS scans. And we are planning to add it soon. So it would make sense to test.if you have the right tools to scan with before each scan. Because you don't have the guarantee that a scan ran before it.

Reply:
Where would you place it?

Reply:
Done